src/main/java/com/celements/metatag/MetaTagApi.java

```diff
@@ -0,0 +1,94 @@
+package com.celements.metatag;
+
+import java.util.Map;
+
+import javax.validation.constraints.NotNull;
+
+import com.celements.metatag.enums.ECharset;
+import com.celements.metatag.enums.EHttpEquiv;
+import com.celements.metatag.enums.ENameNonStandard;
+import com.celements.metatag.enums.ENameStandard;
+import com.celements.metatag.enums.EReferrer;
+import com.celements.metatag.enums.ERobot;
+import com.celements.metatag.enums.EViewport;
+import com.celements.metatag.enums.opengraph.EOpenGraph;
+import com.celements.metatag.enums.twitter.ETwitter;
+import com.celements.metatag.enums.twitter.ETwitterCardType;
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableMap;
+
+public final class MetaTagApi {
```

Comment:
:-1:  MetaTagService
xxxApi are objects which might be used only from scripts (velocity, groovy, ...) The xxxApi is always a wrapper/facade concerned to prevent rights escalations.

It could be that this should be a ScriptService implementation. This would mean that the skripting way to access this singleton is services.xxx . In contrast to the MetaTagApi object being returned by a different ScriptService call.